nifi-docker/dockerhub/README.md

```diff
@@ -95,6 +103,7 @@ Single User Authentication credentials can be specified using environment variab
       -e SINGLE_USER_CREDENTIALS_PASSWORD=ctsBtRBKHRAx69EqUghvvgEvjnaLjFEB \
       apache/nifi:latest
 
+Please note that the password must be 12 chars minimmum, or a new one will be generated (see above to find it in the logs)
```

Comment:
The wording should be adjusted to correct the spelling of `minimum`, and spelling out `characters` would be helpful.
```suggestion
Please note that the password must be 12 characters minimum, otherwise NiFi will generate a random username and password.
```